docs/src/user-guide/configuring/configuration-files-new.md

```diff
@@ -265,7 +267,7 @@ export default [
 ];
 ```
 
-This configuration ensures that the Babel parser, rather than the default, will be used to parse all files ending with `.js` and `.mjs`.
+This configuration ensures that the Babel parser, rather than the default Espree parser, is used to parse all files ending with `.js` and `.mjs`.
```

Comment:
```suggestion
This configuration ensures that the Babel parser, rather than the default Espree parser is used to parse all files ending with `.js` and `.mjs`.
```

Reply:
it is correct to keep this comma here. "rather than the default Espree parser" is a [non-essential aside](https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_with_nonessential_elements.html), which requires using a comma.